### PR TITLE
Making the exception_map work for extensions

### PR DESCRIPTION
**Pull request by hub-cap**
- Added back in the exception_map to the extensions
- Added UnprocessableEntity to the mysql service

### CODE REVIEW

File: reddwarf/extensions/mysql/service.py [hunk 1]
```diff
@@ -34,6 +34,7 @@ class BaseController(wsgi.Controller):
     exclude_attr = []
     exception_map = {
         webob.exc.HTTPUnprocessableEntity: [
+            exception.UnprocessableEntity,
```
Comment (TimSimpsonR) [inline reddwarf/extensions/mysql/service.py:37]: I'm confused what this solved. If UnprocessableEntity is-a exc.HTTPUnprocessableEntity according to inheritance then why wouldn't this already work? The integration tests looking for the correct UnprocessableEntity status code are passing.

Comment (hub-cap) [inline reddwarf/extensions/mysql/service.py:37]: In this case u are right. The map translates our exceptions to webob exceptions so we don't have to inherit from them in all our exceptions. We can remove that inheritance in our exception and it'll still translate.

Comment (TimSimpsonR) [inline reddwarf/extensions/mysql/service.py:37]: Hmm, I suppose if we remove the inheritance the exceptions would be "view / controller" agnostic in that they wouldn't be relying on something from webob. Sounds like a good idea.

**Comment by TimSimpsonR** [comment]
+1

**Comment by hub-cap** [comment]
I won't merge till I remove the inheritance.

**Pull request merged**